js/code.js

```diff
@@ -202,6 +233,8 @@ function addContact()
 
 	document.getElementById("contactAddResult").innerHTML = "";
 
+	justReadCookie();
+
```

Comment:
Uses new justReadCookie() function in order to retrieve current User's ID so we know to who's account to save the Added Contact to.